.vaunt/add-tidb-docs-dash-badge.md

```diff
@@ -0,0 +1,70 @@
+# How to Add TiDB Docs Dash Badges to your GitHub Profile
+
+Your [GitHub profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile) is more than just a collection of repositories; it is your professional identity in the coding world.
+
+This guide walks you through the steps of adding [TiDB Docs Dash 2024](https://www.pingcap.com/event/tidb-docs-dash/) badges to your GitHub profile.
+
+## Step 1. Create a repository for your profile README
+
+If you have already set up your GitHub profile README, skip the following and go to [Step 2. Edit your profile README](#step-2-edit-your-profile-readme).
+
+1. In the upper-right corner of any GitHub page, click **+**, and then click **New repository**.
+
+    <img src="https://docs.github.com/assets/cb-34248/mw-1440/images/help/repository/repo-create-global-nav-update.webp" width="350" />
+
+2. Type a repository name that matches your GitHub username. For example, if your username is `ilovetidb`, the repository name must be `ilovetidb`.
+3. Choose **Public** for the repository visibility.
+4. Select **Initialize this repository with a README file**.
+5. Click **Create repository**.
+
+## Step 2. Edit your profile README
+
+1. On the GitHub page of your `<github_username>` repository, click **Edit README** above the right sidebar.
+2. Add the following code to your README file, replace `{{github_username}}` with your username, and commit the changes directly.
+
+    ```HTML
+    <p>
+      <img src="https://api.vaunt.dev/v1/github/entities/{{github_username}}/achievements?format=svg&limit=3" width="350" />
+    </p>
+    ```
+
+ Your GitHub profile now displays your TiDB Docs Dash badge. For more information about managing your profile README, see [GitHub docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme).
+
+## (Optional) Explore more with Vaunt
+
+TiDB Docs Dash badges are granted via [Vaunt](https://vaunt.dev/), a developer relations platform that aims to empower and grow developer communities.
+
+You can also use Vaunt to showcase your contributions to open-source projects as follows:
```

Comment:
@jeff1010322, I am not sure whether the Vaunt app is a prerequisite for integrating the developer card and community boards. Would you please help confirm? If yes, we also need to indicate that information here.

Reply:
Maybe we can rephrase this line 37 to something like "You can use additional Vaunt features to showcase your contributions to other open source projects as follows:"?

Reply:
I have updated this line as suggested.